view/lib/i18n/locales/ml.json

```
@@ -0,0 +1,1352 @@
+        "2faFailed": "അസാധുവായ 2FA കോഡ്"
+      },
+      "2fa": {
+        "title": "രണ്ടാം ഘട്ട സ്ഥിരീകരണം",
+        "description": "ദയവായി നിങ്ങളുടെ സ്ഥിരീകരണ കോഡ് നൽകുക",
+        "enterCode": "സ്ഥിരീകരണ കോഡ്",
+        "codePlaceholder": "6-അക്ഷര കോഡ് നൽകുക",
+        "verifyButton": "സ്ഥിരീകരിക്കുക",
+        "verifying": "സ്ഥിരീകരിക്കുന്നു..."
+      }
+    },
+    "register": {
+      "title": "രജിസ്റ്റർ ചെയ്യുക",
+      "description": "ഒരു പുതിയ അക്കൗണ്ട് സൃഷ്ടിക്കുക",
+      "email": "ഇമെയിൽ",
+      "password": "രഹസ്യവാക്ക്",
+      "confirmPassword": "രഹസ്യവാക്ക് സ്ഥിരീകരിക്കുക",
+      "submit": "രജിസ്റ്റർ ചെയ്യുക",
+      "emailPlaceholder": "m@example.com",
+      "loading": "ലോഡ് ചെയ്യുന്നു...",
+      "terms": "തുടരുക ക്ലിക്ക് ചെയ്യുന്നതിലൂടെ, നിങ്ങൾ ഞങ്ങളുടെ",
+      "termsOfService": "സേവന നിബന്ധനകൾ",
+      "and": "ഒപ്പം",
+      "privacyPolicy": "സ്വകാര്യതാ നയം",
+      "alreadyHaveAccount": "ഇതിനകം ഒരു അക്കൗണ്ട് ഉണ്ടോ?",
+      "success": "രജിസ്റ്റർ ചെയ്യൽ വിജയകരമാണ്! നിങ്ങളുടെ അക്കൗണ്ട് സ്ഥിരീകരിക്കാൻ ദയവായി നിങ്ങളുടെ ഇമെയിൽ പരിശോധിക്കുക.",
+      "errors": {
+        "invalidEmail": "അസാധുവായ ഇമെയിൽ",
+        "passwordMismatch": "രഹസ്യവാക്കുകൾ പൊരുത്തപ്പെടുന്നില്ല",
+        "passwordRequirements": {
+          "minLength": "പാസ്‌വേഡിൽ കുറഞ്ഞത് 8 പ്രതീകങ്ങളെങ്കിലും ഉണ്ടായിരിക്കണം",
+          "uppercase": "പാസ്‌വേഡിൽ കുറഞ്ഞത് ഒരു വലിയക്ഷരമെങ്കിലും ഉണ്ടായിരിക്കണം",
+          "lowercase": "പാസ്‌വേഡിൽ കുറഞ്ഞത് ഒരു ചെറിയക്ഷരമെങ്കിലും ഉണ്ടായിരിക്കണം",
+          "number": "പാസ്‌വേഡിൽ കുറഞ്ഞത് ഒരു സംഖ്യയെങ്കിലും ഉണ്ടായിരിക്കണം",
+          "specialChar": "പാസ്‌വേഡിൽ കുറഞ്ഞത് ഒരു പ്രത്യേക പ്രതീകമെങ്കിലും ഉണ്ടായിരിക്കണം"
+        },
+        "adminAlreadyRegistered": "അഡ്മിൻ ഇതിനകം രജിസ്റ്റർ ചെയ്തിട്ടുണ്ട്",
+        "adminAlreadyRegisteredDescription": "ദയവായി പ്രവേശനം നേടാൻ അഡ്മിനിസ്ട്രേറ്ററുമായി ബന്ധപ്പെടുക.",
+        "loginButton": "ലോഗിൻ",
+        "somethingWentWrong": "എന്തോ തെറ്റായി പോയി",
+        "loadingAdminRegistration": "അഡ്മിൻ രജിസ്റ്റർ ചെയ്തിട്ടുണ്ടോ എന്ന് പരിശോധിക്കുന്നു...",
+        "tryAgain": "വീണ്ടും ശ്രമിക്കുക",
+        "registerFailed": "രജിസ്റ്റർ ചെയ്യുന്നതിൽ പരാജയപ്പെട്ടു"
+      }
+    },
+    "email": "ഇമെയിൽ",
+    "password": "പാസ്‌വേഡ്",
+    "resetPassword": {
+      "title": "പാസ്‌വേഡ് പുനഃസജ്ജമാക്കുക",
+      "description": "ദയവായി താഴെ നിങ്ങളുടെ പുതിയ പാസ്‌വേഡ് നൽകുക.",
+      "newPassword": "പുതിയ പാസ്‌വേഡ്",
+      "confirmPassword": "പാസ്‌വേഡ് സ്ഥിരീകരിക്കുക",
+      "submit": "പാസ്‌വേഡ് പുനഃസജ്ജമാക്കുക",
+      "submitting": "പാസ്‌വേഡ് പുനഃസജ്ജമാക്കുന്നു...",
+      "success": "പാസ്‌വേഡ് പുനഃസജ്ജമാക്കൽ വിജയകരമാണ്",
+      "errors": {
+        "invalidLink": "അസാധുവായ പുനഃസജ്ജമാക്കൽ ലിങ്ക്",
+        "resetFailed": "പാസ്‌വേഡ് പുനഃസജ്ജമാക്കുന്നതിൽ പരാജയപ്പെട്ടു",
+        "passwordMismatch": "പാസ്‌വേഡുകൾ പൊരുത്തപ്പെടുന്നില്ല",
+        "passwordRequirements": {
+          "minLength": "പാസ്‌വേഡിൽ കുറഞ്ഞത് 8 പ്രതീകങ്ങളെങ്കിലും ഉണ്ടായിരിക്കണം",
```

Example: In the version you have compiled, it is very verbose 

```suggestion
          "minLength": "പാസ്‌വേഡിന് കുറഞ്ഞത് 8 പ്രതീകങ്ങൾ വേണം",
```

This could much simplified version, means the same.